qb/qdelete/delete_exec.go

```diff
@@ -6,13 +6,20 @@ import (
 	"github.com/scylladb/gocqlx/qb"
 
 	"github.com/Drafteame/cassandra-builder/qb/query"
+	"github.com/Drafteame/cassandra-builder/runner"
 )
 
 // Exec execute delete query and return error on failure
 func (dq *Query) Exec() error {
+	runner := runner.New(dq.client)
```

Comment:
sigiero cambiar el nombre para que no colicione con el nombre de paquete:

```suggestion
	run := runner.New(dq.client)
```